octoprint_prometheus_exporter/__init__.py

```diff
@@ -175,6 +182,18 @@ def on_event(self, event, payload):
 	extrusion_total = Counter('octoprint_extrusion_total', 'Filament extruded total')
 	extrusion_print = Gauge('octoprint_extrusion_print', 'Filament extruded this print')
 
+	# X TRAVEL
```

Comment:
We have a hook function, which is fired after a print is finished/failed/canceled. I think we should reset, or unsubscribe the travel_print variables there.

Reply:
I followed how the extrusion counter works. I think they should all behave similar. Wdyt?